bpo-35733: `isinstance(ast.Constant(True), ast.Num)` is `False` (#11547)
python/cpython: Misc/NEWS.d/next/Library/2019-01-13-18-42-41.bpo-35733.eFfLiv.rst

> ``isinstance(ast.Constant(True), ast.Num)`` is ``False``.  Patch by Anthony

Reviewer: ```suggestion
``ast.Constant(boolean)`` no longer an instance of :class:`ast.Num`.  Patch by Anthony
```

> Sottile

Reviewer: ```suggestion
Sottile.
```
